src/lib.rs

```diff
@@ -423,6 +426,48 @@ impl Account {
         })
     }
 
+    /// Load a new account by private key, with a default or custom HTTP client
+    ///
+    /// https://www.rfc-editor.org/rfc/rfc8555#section-7.3.1
+    ///
+    /// the `DefaultClient::try_new()?` can be used as the http client by default.
```

Comment:
similar to my prev comment, for this fn (if both are kept):

```suggestion
    /// the `DefaultClient::try_new()?` will be used as the http client.
```